### ISSUE TITLE
system.do_after problem

### ISSUE THREAD

**Issue by Evgencheg** [Simple-Station/Einstein-Engines#952]
2024-09-22T22:13:55.5486506+03:00 [ERRO] system.do_after: Attempted to cancel do after with an invalid id (52) on entity Фэйт Филдс (54456/n54456, MobHuman, MI13)

same as others, idk how to reproduce... sooo just maybe someone knows

**Comment by VMSolidus** [comment]
I don't think this is technically a bug that can be solved? The entity ceased existing in the middle of a doAfter. and this isn't a fatal error, just a notice of that fact.
